llvm/include/llvm/Analysis/TypeMetadataUtils.h

```diff
@@ -14,6 +14,7 @@
 #ifndef LLVM_ANALYSIS_TYPEMETADATAUTILS_H
 #define LLVM_ANALYSIS_TYPEMETADATAUTILS_H
 
+#include "llvm/IR/GlobalVariable.h"
```

Comment:
Can you instead just use a forward declaration for GlobalVariable, like we do for other types below?